x/swagger/swagger_types_global.go

```diff
@@ -13,13 +13,16 @@ import "github.com/ory/herodot"
 //
 //nolint:deadcode,unused
 //lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
-type errorGeneric struct {
+type ErrorGeneric struct {
 	// Contains error details
 	//
 	// required: true
-	Error herodot.DefaultError `json:"error"`
+	Error GenericError `json:"error"`
 }
 
+// swagger:model genericError
```

Comment:
We now have `errorGeneric` and `genericError` as types in our SDK, I don't think that's desired. How about naming this

```
errorDetails
```

or something like that?

Reply:
Yup, it's weird. But this is exactly how it was before this change https://github.com/ory/herodot/pull/106 (which was done months ago but never rolled out). Any name other than `genericError` for this type will be a much larger change in the generated specs/code.

Reply:
I see, too bad :(